helmclienttest/helmclient.go

```diff
@@ -3,27 +3,31 @@ package helmclienttest
 import (
 	"context"
 
-	"github.com/giantswarm/helmclient"
 	"k8s.io/helm/pkg/helm"
+
+	"github.com/giantswarm/helmclient"
 )
 
 type Config struct {
+	DefaultChart          *helmclient.Chart
+	DefaultError          error
```

Comment:
How about making that:

```
	LoadChartResponse Chart
	LoadChartError    error
```

and deprecating fields below?

Not as flexible as having `LoadChartFunc` but simpler to setup and fits better what we have now.

Reply:
Done.

Reply:
Cool I think `DefaultChart` can be removed now.

Reply:
Yup missed that. Gone now thx!